pkg/operator/ceph/pool/controller.go

```diff
@@ -542,3 +546,25 @@ func (r *ReconcileCephBlockPool) disableMirroring(pool string) error {
 
 	return nil
 }
+
+func (r *ReconcileCephBlockPool) checkRadosNamesapceMirroring(poolName string) (bool, error) {
+	list, err := cephclient.ListRadosNamesapceInPool(r.context, r.clusterInfo, poolName)
+	if err != nil {
+		return false, errors.Wrapf(err, "failed to list rados namespace in pool %q", poolName)
+	}
+
+	isMirroringEnabled := false
+	for _, namespace := range list {
+		poolAndRadosNamespaceName := fmt.Sprintf("%s/%s", poolName, namespace)
+		mirrorInfo, err := cephclient.GetPoolMirroringInfo(r.context, r.clusterInfo, poolAndRadosNamespaceName)
+		if err != nil {
+			return false, errors.Wrapf(err, "failed to get mirroring info for the rados namespace %q", poolAndRadosNamespaceName)
+		}
+		if mirrorInfo.Mode != "disabled" {
+			isMirroringEnabled = true
```

Comment:
Just as well return immediately, since no other side effect in the method
```suggestion
			return true
```